src/lv_conf_kconfig.h

```diff
@@ -10,35 +10,24 @@
 extern "C" {
 #endif
 
+#if defined ESP_PLATFORM
+#include "sdkconfig.h"
+#include "esp_attr.h"
+#endif
+
 /*******************
  * THEME SELECTION
  *******************/
  
 #ifndef LV_THEME_DEFAULT_INIT
  #if defined (CONFIG_LV_THEME_DEFAULT_INIT_EMPTY)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_empty_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_empty_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_TEMPLATE)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_template_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_template_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_MATERIAL)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_material_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_material_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_MONO)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_mono_init
```

Comment:
This line also needs to be changed, isn't it?

Reply:
It's changed below, see my other comment.